docs/overview.md

```diff
@@ -8,16 +7,15 @@ Mesa is a modular framework for building, analyzing and visualizing agent-based
 
 Mesa is modular, meaning that its modeling, analysis and visualization components are kept separate but intended to work together. The modules are grouped into three categories:
 
-1. **Modeling:** Modules used to build the models themselves: a model and agent classes, a scheduler to determine the sequence in which the agents act, and space for them to move around on.
+1. **Modeling:** Modules used to build the models themselves: a model and agent classes, space for them to move around on, and built-in functionality for managing agents.
 2. **Analysis:** Tools to collect data generated from your model, or to run it multiple times with different parameter values.
-3. **Visualization:** Classes to create and launch an interactive model visualization, using a server with a JavaScript interface.
+3. **Visualization:** Classes to create and launch an interactive model visualization, using a browser-based interface.
 
 ### Modeling modules
 
-Most models consist of one class to represent the model itself; one class (or more) for agents; a scheduler to handle time (what order the agents act in), and possibly a space for the agents to inhabit and move through. These are implemented in Mesa's modeling modules:
+Most models consist of one class to represent the model itself and one or more classes for agents. Mesa provides built-in functionality for managing agents and their interactions. These are implemented in Mesa's modeling modules:
```

Comment:
There is no Mesa protocol for agent interactions yet. In NetLogo, you can send a message to another agent:
```
to send-message-to-specific-turtle
  ask turtle 0 [
    ask turtle 1 [
      set my-message "This is a direct message from turtle 0!"
    ]
  ]
  ask turtle 1 [ show my-message ]
end
```

Reply:
Okay, what should I change exactly?

Reply:
I recommend leaving it as, by default agents have to interact, (e.g. in the boltzmann wealth model they are randomly interacting with the exchange of money) it just may not be as explicit as asking another agent what to do.

Reply:
I'd say to remove the claim that Mesa has functionality for agent interaction. You can do it in the Boltzmann wealth model, but it is *not provided by Mesa*.

Reply:
I am sorry, but I fail to understand the problem here. Interaction is a defining feature of ABMs. It seems that @rht equates interaction with messages, but I fail to see this link. Take predator-prey: a wolf moves towards where sheep are. This is interaction. Or Schelling segregation: an agent moves based on the "color" of its neighbors and moves towards an empty spot (determined by which other agents occupy cells). These are all examples of interaction, none requiring a message system.

Reply:
You could say that explicit communication via messaging is a subset of "interaction", but even so, in none of the wolf-sheep or Schelling the interaction is specified in Mesa. Rather, it is specified by the user in Python, with various ways of doing it, not dictated by the library. On the other hand, Mesa has an opinion of how you should manage your agents, i.e. via `AgentSet`.

Reply:
I beg to differ: in both Schelling and Wolf-Sheep the MESA provided spaces that structure the interaction. Moreover, to interact with other agents, an agent will use the model or a space to find the other agent.

Reply:
That would be interaction of the agents *with their environment*. The sentence could be intentionally not specific about the details, but direct interaction with other agents via messaging is not covered nor dictated by Mesa at the moment.

Reply:
And I do intend to integrate https://github.com/ox-inet-resilience/py-economicsl into Mesa at some point in the future for agent messaging and inbox system.